src/cart/cartSlice.js

```diff
@@ -0,0 +1,20 @@
+import React  from "react";
+import { createSlice } from "@reduxjs/toolkit";
+
+const cartSlice = createSlice({
```

Comment:
nice... how did you find using this?

Reply:
> nice... how did you find using this?

It's very neat and clearly separates the reducers of individual modules which helps in organizing the state management in better way